tests/workbenches/conftest.py

```diff
@@ -0,0 +1,309 @@
+from __future__ import annotations
+
+import logging
+import time
+import traceback
+from typing import Callable, Generator
+
+import kubernetes.dynamic
+from kubernetes.dynamic import DynamicClient
+
+import ocp_resources.pod
+import ocp_resources.resource
+
+import pytest
+
+
+@pytest.fixture(scope="function")
+def function_resource_manager(admin_client: DynamicClient) -> Generator[KubeResourceManager, None, None]:
+    resource_manager = KubeResourceManager(admin_client)
+    yield resource_manager
+    resource_manager.destroy()
+
+
+class KubeResourceManager:
```

Comment:
please use ocpenshift python wrapper to CRUD resources

Reply:
done